oat-sa/extension-tao-testqti
# Fix/act 443 lock test session

How to test:
Put the following script into `taoQtiTest/scripts/tools/`:
https://gist.github.com/hutnikau/b739300cfef0128897f8a253f97121a4

Enable LockService:
`sudo -u www-data php index.php '\oat\tao\scripts\install\SetUpLockService' --persistence redis`

1. Run the test
2. Put a breakpoint on `\oat\taoQtiTest\models\runner\QtiRunnerServiceContext::getTestSession()::78`
3. Press next button and make sure that process stopped at breakpoint
4. Run the script: `sudo -u www-data php index.php \oat\taoQtiTest\scripts\tools\CloseTestSession <session_id>`

Expexted behavior: script should be waiting until breakpoint is released. After release of breakpoint the script should finish and close the session. Following pressing to the next button should fail with a message which says that test session is closed
